deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -373,6 +373,12 @@ def get_base_images(request):
     provider_list = baseimages_helper.get_all_providers(request)
     cells_list = cells_helper.get_by_provider(request, DEFAULT_PROVIDER)
     arches_list = arches_helper.get_all(request)
+    for image in base_images:
+        tags = baseimages_helper.get_image_tag_by_id(request, image['id'])
+        golden_tags = [ e['tag'] for e in tags ]
+        image['golden_latest'] = True if 'GOLDEN_LATEST' in golden_tags else False
```

Comment:
`image['golden_latest'] =  'GOLDEN_LATEST' in golden_tags`

Reply:
It is easier for me to check true or false in html tmpl file ^^
That's why I passed boolean :)

Reply:
`'GOLDEN_LATEST' in golden_tags` will return a boolean :)

Reply:
oh, I see the point LOL